docs/walletkit/android/chain-abstraction.mdx

```diff
@@ -0,0 +1,233 @@
+# Chain Abstraction
+
+Chain Abstraction in WalletKit enables users with stablecoins on any network to spend them on-the-fly on a different network. Our Chain Abstraction solution provides a toolkit for wallet developers to integrate this complex functionality using WalletKit.
+
+For example, when an app requests a 100 USDC payment on Base network but the user only has USDC on Arbitrum, WalletKit offers methods to detect this mismatch, generate necessary transactions, track the cross-chain transfer, and complete the original transaction after bridging finishes.
+
+:::note
+Chain abstraction is currently in experimental phase and requires the `@ChainAbstractionExperimentalApi` annotation.
+:::
+
+## How It Works
+
+When a dapp requests a transaction through `eth_sendTransaction`, you need to:
+1. Detect if chain abstraction is needed
+2. Generate necessary bridging transactions
+3. Handle transaction execution and monitoring
+4. Complete the original transaction once bridging is done
+
+The following sequence diagram illustrates the complete flow of a chain abstraction operation, from the initial dapp request to the final transaction confirmation
+
+![Chain Abstraction Flow](/assets/chain-abstraction-flow.png)
```

Comment:
The diagram flow has the old method names: canFulfill, prepareFulfillment

Reply:
Updated